<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Chitralekha" failures="0" tests="2" name="com.orangehrm.tests.LoginOrangeHRM" time="0.119" errors="2" timestamp="2025-08-23T17:14:11 IST" skipped="0">
  <testcase classname="com.orangehrm.tests.LoginOrangeHRM" name="setupreport" time="0.119">
    <error message="Cannot invoke &quot;com.aventstack.extentreports.ExtentReports.attachReporter(com.aventstack.extentreports.observer.ExtentObserver[])&quot; because &quot;com.orangehrm.utilities.ExtentManager.extent&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentReports.attachReporter(com.aventstack.extentreports.observer.ExtentObserver[])" because "com.orangehrm.utilities.ExtentManager.extent" is null
at com.orangehrm.utilities.ExtentManager.getinstance(ExtentManager.java:17)
at com.orangehrm.base.BaseTest.setupreport(BaseTest.java:23)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:358)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- setupreport -->
  <testcase classname="com.orangehrm.tests.LoginOrangeHRM" name="verifylogin" time="0.000">
    <error message="
Method public void com.orangehrm.tests.LoginOrangeHRM.verifylogin(java.lang.String,java.lang.String) throws java.io.IOException requires a @DataProvider named : logindate" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Method public void com.orangehrm.tests.LoginOrangeHRM.verifylogin(java.lang.String,java.lang.String) throws java.io.IOException requires a @DataProvider named : logindate
at org.testng.internal.Parameters.findDataProvider(Parameters.java:532)
at org.testng.internal.Parameters.handleParameters(Parameters.java:769)
at org.testng.internal.Parameters.handleParameters(Parameters.java:734)
at org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)
at org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:949)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- verifylogin -->
</testsuite> <!-- com.orangehrm.tests.LoginOrangeHRM -->
